commands/remove-container.ts

```diff
@@ -0,0 +1,55 @@
+import vscode = require('vscode');
+import { docker } from './utils/docker-endpoint';
+import { reporter } from '../telemetry/telemetry';
+import { DockerNode } from '../explorer/dockerExplorer';
+import { dockerExplorerProvider } from '../dockerExtension';
+import { ContainerItem, quickPickContainer } from './utils/quick-pick-container';
+
+const teleCmdId: string = 'vscode-docker.container.remove';
+
+export async function removeContainer(context?: DockerNode) {
+
+    let containersToRemove: Docker.ContainerDesc[];
+
+    if (context && context.containerDesc) {
+        containersToRemove = [context.containerDesc];
+    } else {
+        const opts = {
+            "filters": {
+                "status": ["created", "restarting", "running", "paused", "exited", "dead"]
```

Comment:
It looks like the `options` argument of `quickPickContainer` is only being used to specify a container filter. Would it simpler to have it take a bool that indicates whether to display all containers or just the running ones? Then the commands that display the quick pick don't need to worry about defining this filter object.